ReactCommon/hermes/inspector/Inspector.cpp

```diff
@@ -32,6 +32,7 @@ namespace futures {
 
 SemiFuture<Unit> sleep(Duration, Timekeeper *) {
   LOG(FATAL) << "folly::futures::sleep() not implemented";
+  return SemiFuture<Unit>();
```

Comment:
LOG(FATAL) does not return.  I guess glog doesn't do the appropriate declarations on all windows environments.  Can we use DECLSPEC_NORETURN or [[noreturn]] or someting on the function, instead?  Same below.

Reply:
Actually, I just read your changelog.  If you're stubbing out glog, then the code in the PR is dangerous, as returning nullptr below will likely crash.  And declaring it noreturn is incorrect, as it will return.  Is it possible to do something in your environment which obeys the original intent, which is to fail with a useful error if this code is called unexpectedly?  You could do this by making the glog stubs a bit smarter, or just writing some code conditional on _MSC_VER in here.


  Although it should never happen (hence the use of LOG(FATAL)), if it does, this will certainly crash.  It seems like there should be a more explicit way to handle